### PR TITLE
Feature/change password page UI

### PR DESCRIPTION
**Pull request by ganeshmurali-dev**
1. Back to login extra arrow button 
2. Inputs focus and error message
3. Popover UI and Icons fonts

It was all fixed

### CODE REVIEW

File: src/components/PasswordRequirement/PasswordRequirement.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import React from "react";
+
+import "./scss/PasswordRequirement.css";
+export const passwordRequirements = (password) => {
+  const lengthCheck = /.{8,}/.test(password);
+  const upperCaseCheck = /[A-Z]/.test(password);
+  const lowerCaseCheck = /[a-z]/.test(password);
```
Comment (avinashdckap) [inline src/components/PasswordRequirement/PasswordRequirement.jsx]: Make this as utility function which has switch case statement, return the error from that function 
Exxample: 
passwordRequirement()=> returns "Minimum 8 characters" and errorState
Based on that handle in JSX

File: src/components/PasswordRequirement/PasswordRequirement.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+          <span>
+            {" "}
+            <img className="close-btn" src="/icons/remove.svg" alt="cancel" />
+          </span>
+        )}{" "}
+        At least one special character
+      </li>
+    </ul>
+  );
+  setpasswordCriteria(content);
```
Comment (avinashdckap) [inline src/components/PasswordRequirement/PasswordRequirement.jsx]: Wrong, you can't setstate in JSX

File: src/pages/changePasswordPage/ChangePasswordPage.jsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+import { checkPasswordCriteria } from "../../components/PasswordRequirement/PasswordRequirement";
+
+//CSS here
+import "./scss/ChangePasswordPage.css";
+const ResetPasswordPage = () => {
+  const [passwordError, setPasswordError] = useState({});
+  const [passwordCriteria, setpasswordCriteria] = useState(null);
+  const [popoverShow, setPopovershow] = useState(false);
+  const [newPassword, setNewPassword] = useState({
+    newPassword: "",
```
Comment (avinashdckap) [inline src/pages/changePasswordPage/ChangePasswordPage.jsx]: I have told not to use an object for state on my previous comments

**Review by avinashdckap — CHANGES_REQUESTED** [review]

**Comment by avinashdckap** [comment]
Screenshot from 2024-01-08 10-23-05

Why error showing only on one field?

**Comment by avinashdckap** [comment]
Your password must be at least 8 characters long. Avoid common words or patterns.
Why this content is shown? we're showing the same on popover also!

File: src/components/PasswordRequirement/PasswordRequirement.jsx (hunk 1 above)
Comment (ganeshmurali-dev) [reply src/components/PasswordRequirement/PasswordRequirement.jsx]: Changed to switch case

File: src/components/PasswordRequirement/PasswordRequirement.jsx (hunk 2 above)
Comment (ganeshmurali-dev) [reply src/components/PasswordRequirement/PasswordRequirement.jsx]: changed.

File: src/pages/changePasswordPage/ChangePasswordPage.jsx (hunk 3 above)
Comment (ganeshmurali-dev) [reply src/pages/changePasswordPage/ChangePasswordPage.jsx]: I have changed from object to separate state

**Comment by ganeshmurali-dev** [comment]
1. Password icon as in integrator
Changed in login page and change password page.

2. I changed the switch case in PasswordRequirements page

**Pull request merged**